### PR TITLE
Channel layout Update and Responsive + NFT

### PR DESCRIPTION
**Pull request by eldiegod**
fixes: #1856

### CODE REVIEW

File: src/components/Avatar/Avatar.styles.ts
Comment (drillprop) [inline src/components/Avatar/Avatar.styles.ts]: Hmm, I'm a little worried that this change could break avatar somewhere else. Are you sure that we need this? Maybe we could use `useMediaMatch`  inside component instead.

File: src/views/viewer/ChannelView/ChannelView.styles.ts
Comment (drillprop) [inline src/views/viewer/ChannelView/ChannelView.styles.ts]: I don't see the point of creating this, let's just use [Collectors panel](https://github.com/Joystream/atlas/pull/1948) once it's merged. I would just remove this and not include this in this PR. I know that my PR is blocking you, but I think we could include finished CollectorsBox in a separate PR.

File: src/views/viewer/ChannelView/ChannelView.tsx
Comment (drillprop) [inline src/views/viewer/ChannelView/ChannelView.tsx]: Let's remove this as well

File: src/views/viewer/ChannelView/ChannelView.tsx
Comment (drillprop) [inline src/views/viewer/ChannelView/ChannelView.tsx]:
```suggestion
```
Let's just delete this. Also, CollectorsBoxContainer doesn't work well, If you remove `CollectorsBoxContainer`(If I'm not mistaken, some channels will not have collectorsBox), the layout will be broken.
Look: 
image

File: src/views/viewer/ChannelView/ChannelView.tsx
Comment (drillprop) [inline src/views/viewer/ChannelView/ChannelView.tsx]: let's just delete this as well for now.

**Review by drillprop — CHANGES_REQUESTED** [review]

File: src/components/Avatar/Avatar.styles.ts
Comment (eldiegod) [reply src/components/Avatar/Avatar.styles.ts]: I think the api decision might be inconsistent with the other avatar sizes, but as the variable name suggests, this is only to be used for the `channelAvatar`. I also would like this to be separate implementation but at the moment is a variant. 

Not too sure what to do here.

Comment (drillprop) [reply src/components/Avatar/Avatar.styles.ts]: Did you check studio views as well? if you're sure that it doesn't break anything, I'm fine with that change 👍🏻

**Comment by drillprop** [comment]
Just took a glance at the studio. Your PR is breaking spacing on `studio/channel`. 
image
should be: 
image

File: src/components/Avatar/Avatar.styles.ts
Comment (eldiegod) [reply src/components/Avatar/Avatar.styles.ts]: It does break sadly.

Should we just keep it consistent with the variant or what do we do here @toiletgranny ?

File: packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx
Comment (drillprop) [inline packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx]: If there are no collectors we will see: 
image

And if I remove CollectorsBox there will be a problem with spacing: 
image

**Review by drillprop — CHANGES_REQUESTED** [review]

File: packages/atlas/src/views/studio/CreateEditChannelView/CreateEditChannelView.styles.ts
Comment (drillprop) [inline packages/atlas/src/views/studio/CreateEditChannelView/CreateEditChannelView.styles.ts]: I'm pretty sure, that we don't need to add `!important` here.

File: packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts
Comment (drillprop) [inline packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts]: Why `193.333`? Where does it come from?

File: packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx
Comment (drillprop) [inline packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx:252]: I still don't like the fact we have this empty div that just taking the space 😅 
image
I mean, I don't have a better solution for this, so if you don't come up with an idea of how to remove this guy, just leave it as it is.

**Review by drillprop — CHANGES_REQUESTED** [review]

File: packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx
Comment (eldiegod) [reply packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx:252]: I don't have an issue with it it just keeps the space for the collector box, a solution for it would be more complicated than just not doing nothing

File: packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts
Comment (eldiegod) [reply packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts]: 250px was the size of the sort select + label but they removed the label so 193.333 is the remaining size the select should have without it

File: src/components/Avatar/Avatar.styles.ts
Comment (toiletgranny) [reply src/components/Avatar/Avatar.styles.ts]: Talked to @Lenorette, waiting for her response.

File: packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts
Comment (drillprop) [reply packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts]: This looks odd, and I still don't see the point of setting this to exactly 193.333 px.  
Why can't we just set this to a rounded value like 200px?

**Comment by eldiegod** [comment]
Had a chat with Adam from design and decided to keep the Avatar variant as it is for now.

**Comment by Lenorette** [comment]
Hey @eldiegod I've talked to Adam and I've changed avatar size on 768px breakpoint to 88px. 
https://www.figma.com/file/POw4j95jszfIP5bGfas2IC/Channel-page?node-id=510%3A24776

I did the same changes also on member profile, so it's the same everywhere 
https://www.figma.com/file/yhZpTHdf1sxJx13uRZ71GV/Membership-profile?node-id=923%3A136157

Let me know if its ok now 🙌

**Comment by eldiegod** [comment]
> Let me know if its ok now 🙌

@Lenorette awesome, yes its great, thanks a lot for the quickness ⚡️

File: packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts
Comment (eldiegod) [reply packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts]: @drillprop So, made it 160px as per design, as it no longer has a value label

File: packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx
Comment (drillprop) [inline packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx:226]: Let's maybe set this as an empty array for now and add the `todo` comment here? 
I'm a little worried that we deploy this by accident.

**Review by drillprop — APPROVED** [review]

File: packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx
Comment (kdembler) [reply packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx:226]: Certainly! @eldiegod for now please let's hide this section and also the NFTs tab 🙏

Comment (eldiegod) [reply packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx:226]: its done @kdembler @drillprop

**Pull request merged**
